bin/update_cucumber

```diff
@@ -100,6 +105,9 @@ class {
         ];
     }
 
+    /**
+     * @return array<array-key, mixed>
```

Comment:
What does this provide over the native PHP `: array` return type?

Reply:
I've enabled PHPStan strict checks, which disallows just using `array`.

The reason is that while in some cases (like that one) there is no advantage, PHPStan cannot catch cases where there should have been a better better/detailed type hint.